src/org/labkey/test/components/glassLibrary/grids/Pager.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation. All rights reserved. No portion of this work may be reproduced in
+ * any form or by any electronic or mechanical means without written permission from LabKey Corporation.
+ */
+package org.labkey.test.components.glassLibrary.grids;
+
+import org.labkey.test.Locator;
+import org.labkey.test.components.WebDriverComponent;
+import org.labkey.test.components.react.DropdownButtonGroup;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class Pager extends WebDriverComponent<Pager.ElementCache>
+{
+    private final WebElement _el;
+    private final WebDriver _driver;
+    //private final ResponsiveGrid _grid;
+
+    protected Pager(WebElement element, WebDriver driver)
+    {
+        _el = element;
+        _driver = driver;
+    }
+
+    @Override
+    public WebElement getComponentElement()
+    {
+        return _el;
+    }
+
+    @Override
+    public WebDriver getDriver()
+    {
+        return _driver;
+    }
+
+    public Pager jumpToPage(String jumpTo)
+    {
+        elementCache().jumpToDropdown.clickSubMenu(jumpTo);
+        return this;
+    }
+    public int getCurrentPage()
+    {
+        return Integer.parseInt(elementCache().jumpToDropdown.getButtonText());
+    }
+
+    public Pager selectPageSize(String pageSize)
+    {
+        elementCache().pageSizeDropdown.clickSubMenu(pageSize);
+        return this;
+    }
+
+    public int getPageSize()
+    {
+        return Integer.parseInt(elementCache().pageSizeDropdown.getButtonText());
+    }
+
+    public Pager clickPrevious()
+    {
+        elementCache().prevButton.click();
+        return this;
+    }
+
+    public boolean isPreviousEnabled()
+    {
+        return !elementCache().prevButton.getAttribute("disabled").equals("true");
+    }
+
+    public Pager clickNextButton()
```

Comment:
Use consistent method names
```suggestion
    public Pager clickNext()
```

Reply:
done